plenigo_js/tinymce-plenigo-plugin.js

```diff
@@ -33,8 +43,19 @@ tinymce.PluginManager.add('plenigo', function (editor, url) {
                     var target_text = '';
                     var affiliate_text = '';
                     var class_text = '';
+                    var price_text = '';
+                    var isInvalid = false;
+                    var price = e.data.price.trim();
+                    var quantity = e.data.quantity.trim();
```

Comment:
null check of quantity

Reply:
the data by default is an empty string when it is not filled, this is why i don't do null checks